gtest/src/manager.rs

```diff
@@ -30,17 +30,70 @@ use gear_core::{
 use std::collections::{BTreeMap, VecDeque};
 use std::time::{SystemTime, UNIX_EPOCH};
 
-#[derive(Clone, Debug)]
-pub(crate) enum ProgramState {
-    Initialized,
-    Uninitialized(Option<MessageId>),
-    FailedInitialization,
+#[derive(Debug)]
+pub(crate) enum Actor {
```

Comment:
If we go with pallet-style, isn't this seems better to separate ProgramState for other enum variants?